com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -498,6 +500,18 @@ public void UpdateConnectionsCount(int count)
 #endif
         }
 
+        public void UpdatePacketLoss(float count)
```

Comment:
What does count mean in this case? In your other PR you mentioned this was a percentage 🤔

Reply:
Fixed to be named packetLoss